### ISSUE TITLE
ISO for 3.0.2 release fails to boot with kernel panic error

### ISSUE THREAD

**Issue by PROBLEMCHYLD** [redorescue/redorescue#53]
I get a kernel panic with 3.0.2. I replace the initrd and vmlinuz from 3.0.1 and things are working. Something is broken with 3.0.2 initrd and vmlinuz. 3.0.1 works great....

**Comment by zebradots** [comment]
Yes, something went wrong here, possibly with the transfer of the ISO or the build itself. With only 72 downloads so far, we've deleted the ISO file and will replace it with an updated version under the same name as soon as it has been corrected (along with an updated `SHA256SUMS` file).

**Bad sha256 hash:** `98c4421c3542d4585ddb071510661bcbba968f3ea285939ac8a5cf76e346db1e`

We'll confirm the new checksum once the new ISO is released.
